src/ReactotronConfig.js

```diff
@@ -1,20 +1,21 @@
 /* eslint-disable */
-import Reactotron, { asyncStorage, networking, openInEditor } from 'reactotron-react-native';
-import { NativeModules } from 'react-native';
+if (__DEV__) {
```

Comment:
Moved all the Required code into `__DEV__` flag so that none of the Reactotron related code gets executed in the production build. This makes zero impact on production build.